### PR TITLE
Simplify SuiteController::execute method logic

### PR DESCRIPTION
**Pull request by spolischook**

### CODE REVIEW

File: src/Behat/Testwork/Suite/Cli/SuiteController.php [hunk 1]
```diff
@@ -60,24 +60,22 @@ public function configure(Command $command)
      * {@inheritdoc}
      */
     public function execute(InputInterface $input, OutputInterface $output)
-    {
-        $exerciseSuiteName = $input->getOption('suite');
+    {
```
Comment (stof) [inline src/Behat/Testwork/Suite/Cli/SuiteController.php]: please don't add trailing whitespaces

File: src/Behat/Testwork/Suite/Cli/SuiteController.php [hunk 2]
```diff
@@ -60,24 +60,22 @@ public function configure(Command $command)
      * {@inheritdoc}
      */
     public function execute(InputInterface $input, OutputInterface $output)
-    {
-        $exerciseSuiteName = $input->getOption('suite');
```
Comment (stof) [inline src/Behat/Testwork/Suite/Cli/SuiteController.php]: please keep the variable definition outside the ``if``. It is more readable

File: src/Behat/Testwork/Suite/Cli/SuiteController.php (hunk 1 above)
Comment (spolischook) [inline src/Behat/Testwork/Suite/Cli/SuiteController.php]: fixed

File: src/Behat/Testwork/Suite/Cli/SuiteController.php (hunk 2 above)
Comment (spolischook) [inline src/Behat/Testwork/Suite/Cli/SuiteController.php]: fixed

**Comment by stof** [comment]
This is breaking everything. Previously, when the suite name was not provided, all suites were registered. Now, you register none of them

**Pull request closed without merging**